recipes/panda3d_viewer/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "panda3d_viewer" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 623842303b9f568d80f5a300d35079738f578917916394abaca03ac4fa7c130a
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python 3.6
```

Comment:
Is this py36 only? If not please use `>=` instead.

Reply:
Yes, it should be `>=`, it is wath it was first. But for the PR I fixed it to 3.6 because the C.I. building fail when it does consecutively python 3.6 and 3.7. It does not fail on 3.6 only, nor 3.7 only.
I know that in stage-recipe C.I. the different python build are consecutive whereas in feedstock they are independant, so I was planing to reintroduce `>=` in the feedstock.

Reply:
Those failures may be important then. Let's try it here again and I'll check the logs.